### DISCUSSION TITLE
polygonZkEvmTestnet chain id does not match the official doc

### DISCUSSION THREAD

**Discussion by davidcalap** [wevm/wagmi#1955] [Q&A]
Hi,

I'm using wagmi 0.11.7 and I want to use polygonZkEvmTestnet which is supported based on the doc: https://wagmi.sh/core/chains

In the code I see in the chains module the chain declared as follows:

```
declare const polygonZkEvmTestnet: {
    readonly id: 1422;
    readonly name: "Polygon zkEVM Testnet";
    readonly network: "polygon-zkevm-testnet";
    readonly nativeCurrency: {
        readonly name: "Ether";
        readonly symbol: "ETH";
        readonly decimals: 18;
    };
    readonly rpcUrls: {
        readonly default: {
            readonly http: readonly ["https://rpc.public.zkevm-test.net"];
        };
        readonly public: {
            readonly http: readonly ["https://rpc.public.zkevm-test.net"];
        };
    };
    readonly blockExplorers: {
        readonly default: {
            readonly name: "Blockscout";
            readonly url: "https://explorer.public.zkevm-test.net";
        };
    };
    readonly testnet: true;
};
```

But in the Polygon doc (https://wiki.polygon.technology/docs/zkEVM/develop/#connecting-to-zkevm) I see this:

Network Name: Polygon zkEVM Testnet
RPC URL: https://rpc.public.zkevm-test.net/
Chain ID: 1442
Currency Symbol: ETH
Block Explorer URL: https://explorer.public.zkevm-test.net/

**Note that the chain ids doesn't match (1422 vs 1442)**

Is this a known bug? Could you help us  to fix it?

**Comment by codingwithmanny** [comment]
Proposed the fix in this PR https://github.com/wagmi-dev/references/pull/174

**Reply by joanseg** [reply]
Many thanks 🙏
